ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -79,16 +79,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-edge-zone
-- as: aws-ipi-disc-priv-sno-f28
```

Comment:
Is there any reason to remove this job?

Reply:
All the stable profiles are selected from nightly profiles pools, there is no `aws-ipi-disc-priv-sno` profile any more, so remove it here.